.gitignore

```diff
@@ -27,7 +27,6 @@ var/
 #  Usually these files are written by a python script from a template
 #  before PyInstaller builds the exe, so as to inject date/other infos into it.
 *.manifest
-*.spec
```

Comment:
If I understand this correctly, we have perfectly good reasons to remove this line and supply a `.spec` file with our sources, right? Is this the way to do it, or is the `.spec` file only intended as a temporary intermediate file?